app/components/password_toggle_component.html.erb

```diff
@@ -0,0 +1,25 @@
+<lg-password-toggle class="password-toggle--label-<%= toggle_position %>">
```

Comment:
should we consider factoring this out so it's easier to search for the uses of specific classes? `password-togle--label-top` and `password-togle--label-bottom`

Reply:
> should we consider factoring this out so it's easier to search for the uses of specific classes? password-togle--label-top and password-togle--label-bottom

Good idea! Updated in 397fbb7f2.